syntax = "proto3";

option java_package = "com.huasheng.sa.quant.open.sdk.protobuf.hq.response";
option java_outer_classname = "UsOptionChainCodeResponseProto";

message UsOptionChainCodeResponse {
  repeated string optionCode = 1; //期权代码
}